package.json

```diff
@@ -1,74 +1,25 @@
 {
-  "name": "@react-three/cannon",
-  "version": "4.8.0",
-  "description": "physics based hooks for react-three-fiber",
-  "keywords": [
-    "cannon",
-    "three",
-    "react",
-    "react-three-fiber",
-    "physics"
-  ],
-  "author": "Paul Henschel",
-  "contributors": [
-    "Cody Persinger <codypersinger@gmail.com> (https://github.com/codynova)",
-    "Bjorn Stromberg <bjorn@bjornstar.com> (https://github.com/bjornstar)"
-  ],
-  "repository": "github:pmndrs/use-cannon",
-  "license": "MIT",
-  "module": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "sideEffects": false,
+  "name": "use-cannon",
+  "private": true,
   "scripts": {
-    "build": "rollup -c",
-    "eslint": "eslint .",
-    "eslint-fix": "eslint --fix .",
-    "prebuild": "tsc",
-    "prepare": "husky install",
-    "prettier": "prettier --list-different .",
-    "prettier-fix": "prettier --write .",
-    "test": "echo tests are missing"
-  },
-  "dependencies": {
-    "cannon-es": "^0.18.0",
-    "cannon-es-debugger": "^1.0.0",
-    "events": "^3.3.0"
+    "build": "yarn workspace @react-three/cannon run build",
+    "eslint": "yarn workspaces foreach -v run eslint",
+    "eslint-fix": "yarn workspaces foreach -v run eslint-fix",
+    "prettier": "yarn workspaces foreach -v run prettier",
+    "prettier-fix": "yarn workspaces foreach -v run prettier-fix",
+    "postinstall": "husky install"
   },
+  "packageManager": "yarn@3.2.0",
+  "workspaces": [
+    "packages/*",
+    "examples"
```

Comment:
I think `examples` should be `packages/react-three-cannon-examples`

Reply:
Sure, moved now